docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -134,8 +134,33 @@ function template_preprocess_uiowa_core_headline(&$variables) {
 /**
  * Implements hook_preprocess_HOOK().
  */
-function uiowa_core_preprocess_page(&$variables) {
+function uiowa_core_preprocess_node(&$variables) {
+
+  if ($variables['view_mode'] == 'teaser') {
+
+    $show_visual_indicators_on_teasers = FALSE;
+    $node_type = $variables['node']->bundle();
+
+    switch ($node_type) {
+      case 'page':
+        $show_visual_indicators_on_teasers = \Drupal::configFactory()->getEditable('sitenow_pages.settings')->get('show_visual_indicators_on_teasers');
+        break;
+
+      case 'person':
+        $show_visual_indicators_on_teasers = \Drupal::configFactory()->getEditable('sitenow_people.settings')->get('show_visual_indicators_on_teasers');
+        break;
+    }
```

Comment:
It seems like this could be rewritten similar to the way that we did it in https://github.com/uiowa/uiowa/pull/5226/files

Reply:
Refactored here: https://github.com/uiowa/uiowa/pull/5094/commits/39802b0ba2c3142bdde21ac636ae83f0c46cbf0b